llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -4687,13 +4687,24 @@ InstructionCost AArch64TTIImpl::getPartialReductionCost(
   EVT InputEVT = EVT::getEVT(InputTypeA);
   EVT AccumEVT = EVT::getEVT(AccumType);
 
-  if (VF.isScalable() && !ST->isSVEorStreamingSVEAvailable())
-    return Invalid;
+  unsigned VFMinValue = VF.getKnownMinValue();
+
+  if (VF.isScalable()) {
+    if (!ST->isSVEorStreamingSVEAvailable())
+      return Invalid;
+
+    // Don't accept a partial reduction if the scaled accumulator is vscale x 1,
+    // since we can't lower that type.
+    unsigned Scale =
+        AccumEVT.getScalarSizeInBits() / InputEVT.getScalarSizeInBits();
+    if (VFMinValue / Scale == 1)
```

Comment:
Is this the same as:
```suggestion
    if (VFMinValue == Scale)
```
?

Reply:
It is! Fixed.

Reply:
Hmm, I don't see any update here with the change?

Reply:
Hmm I did push it and according to my local branch it's on my remote branch.

Reply:
https://github.com/SamTebbs33/llvm-project/commit/02734d9c29bac3d1fd26cbc6aa3d7620fab21642